### DISCUSSION TITLE
Experiencing preload error

### DISCUSSION THREAD

**Discussion by kfbfarley** [vercel/community#996] [Help] [answered]
`Error: File /vercel/path1/node_modules/simple-update-notifier/node_modules/semver/preload.js does not exist.`

![image](https://user-images.githubusercontent.com/42237552/197576578-22bcc9de-829b-4a62-aae5-1f0320f3ebef.png)

**Comment by amyegan** [comment, marked as answer]
Hi @kfbfarley. Are you still facing this error?

I did some digging and found that a [similar error](https://github.com/vercel/vercel/discussions/5900#discussioncomment-412730) was solved by deleting the lockfile and `node_modules` directory and then reinstalling to regenerate the file with compatible package versions. You should also be able to do a dry run of the build using [`vercel build`](https://vercel.com/docs/cli/build) from your local computer. Adding the [`--debug` option](https://vercel.com/docs/cli/global-options#debug) will make it print more detail to help with debugging if you get any local errors.
